dist/stat_data.py

```diff
@@ -551,7 +551,10 @@ def __init__(self, name, desc, flags=''):
     TxnStat('txn_prepare_active', 'prepared transactions currently active'),
     TxnStat('txn_prepare_commit', 'prepared transactions committed'),
     TxnStat('txn_prepare_rollback', 'prepared transactions rolled back'),
+    TxnStat('txn_prepared_updates_commit_count', 'Number of prepared updates committed'),
     TxnStat('txn_prepared_updates_count', 'Number of prepared updates'),
+    TxnStat('txn_prepared_updates_key_repeated_count', 'Number of prepared updates repeated on the same key'),
+    TxnStat('txn_prepared_updates_rollback_count', 'Number of prepared updates rollbacked'),
```

Comment:
"Number of prepared updates rolled back"

Reply:
My preference for stat name is "txn_prepared_updates_committed". Word "count" in name is not required. Similarly "txn_prepared_updates", txn_prepared_updates_rolledback"

Reply:
Fixed.